src/frontend/data/sideEffects/getResource/index.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import fetchMock from 'fetch-mock';
+
+import { requestStatus } from '../../../types/api';
+import { modelName } from '../../../types/models';
+import { report } from '../../../utils/errors/report';
+import { videoMockFactory } from '../../../utils/tests/factories';
+import { addResource } from '../../stores/generics';
+import { getResource } from './';
+
+jest.mock('../../appData', () => ({
+  appData: {
+    jwt: 'some token',
+  },
+}));
+
+jest.mock('../../stores/generics', () => ({
+  addResource: jest.fn(),
+}));
+
+jest.mock('../../../utils/errors/report', () => ({
+  report: jest.fn(),
+}));
+
+const mockAddResource = addResource as jest.MockedFunction<typeof addResource>;
```

Comment:
You could declare `mockAddResource` in one line through `jest.spyOn`
```suggestion
jest.mock('../../../utils/errors/report', () => ({
  report: jest.fn(),
}));

const mockAddResource = jest.spyOn(require('../../stores/generics'), 'addResource');
```

Reply:
Should we really use `spyOn` ? I don't really spy the function.